geopyspark-geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala

```diff
@@ -130,6 +130,14 @@ class SpatialTiledRasterLayer(
     SpatialTiledRasterLayer(zoom, tileLayer)
   }
 
+  def addTemporal(temporalKey:TemporalKey): TiledRasterLayer[SpaceTimeKey] = {
```

Comment:
What about the name? 'add' sounds like we simply add something, while we're actually doing a conversion from spatial to temporal.
Does 'asTemporal' sound reasonable? Or are there similar method names that we can align to?